src/tactic/ui/panel/edit_wdg.py

```diff
@@ -277,6 +277,21 @@ def init(my):
         project_code = Project.get_project_code()
 
 
+        if my.parent_key:
+            from pyasm.biz import Schema
+            for element_name in my.element_names:
+                schema = Schema.get()
+                parent_sobject = Search.get_by_search_key(my.parent_key)
+                parent_stype = parent_sobject.get_search_type()
```

Comment:
This is preferred.. since ur way risks NoneType Exception
parent_stype = SearchKey.extract_base_search_type(my.parent_key)